server/storage/mvcc/revision.go

```diff
@@ -55,7 +57,8 @@ func RevToBytes(rev Revision, bytes []byte) []byte {
 }
 
 func BytesToRev(bytes []byte) Revision {
-	return BytesToBucketKey(bytes).Revision
+	rev, _ := BytesToBucketKey(bytes)
```

Comment:
not sure we should swallow this error, your validations definitely have merit in ensuring the byte slice is correct

Reply:
Changing signature fo `mvcc.BytesToRev` propagates above layers and exported interfaces. I prefer to keep the scope of this PR to improving snapshot status and create another PR for handling `mvcc.BytesToRev` errors.

I think it's a valid use case to use this function without checking errors. Maybe we can rename it as `MustBytesToRev` and make it panic on errors?

Reply:
`panic` sounds a lot more reasonable to me than returning rev 0 with a swallowed error. 

> I prefer to keep the scope of this PR to improving snapshot status and create another PR for handling mvcc.BytesToRev errors.

sure, if you want to do the larger refactoring later, then leave it for now :)